internal/telemetry/telemetry.go

```diff
@@ -9,36 +9,34 @@ package telemetry
 
 import (
 	"time"
-
-	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec"
 )
 
-// ProductStart signals that the product has started with some configuration
-// information. It will start the telemetry client if it is not already started.
-// ProductStart assumes that the telemetry client has been configured already by the caller using the ApplyOps method.
-// If the client is already started, it will send any necessary app-product-change
-// events to indicate whether the product is enabled, as well as an app-client-configuration-change
-// event in case any new configuration information is available.
-func (c *client) ProductStart(namespace Namespace, configuration []Configuration) {
+// ProductChange signals that the product has changed with some configuration
+// information. It will start the telemetry client if it is not already started,
+// unless enabled is false (in which case the call does nothing). ProductChange
+// assumes that the telemetry client has been configured already by the caller
+// using the ApplyOps method.
+// If the client is already started, it will send any necessary
+// app-product-change events to indicate whether the product is enabled, as well
+// as an app-client-configuration-change event in case any new configuration
+// information is available.
+func (c *client) ProductChange(namespace Namespace, enabled bool, configuration []Configuration) {
```

Comment:
I'm not seeing any situations in the code where `enabled=false`. Is that coming later, or can we just assume that it will always be true and omit this function parameter?

Reply:
We can actually probably use this to support remote configuration of disabling the tracer, which is planned soon, so this will probably come in handy for that.

Reply:
There's one in `appsec.stop()` -- which happens when AppSec remote activation is reversed (disabled).